### PR TITLE
Adding Dockerfile for CDCgov/tostadas

### PR DESCRIPTION
**Pull request by kyleoconnell**
Dockerfile to accompany the CDC Tostadas pipeline for viral annotation and submission to NCBI databases.
https://github.com/CDCgov/tostadas

Please replace all '[ ]' with '[X]' to demonstrate completion.

Pull Request (PR) checklist:
- [X ] Include a description of what is in this pull request in this message.
- [ X] The dockerfile successfully builds to a test target for the user creating the PR. (Built with `docker build -t tostadas:1.0.0 ./tostadas`)
- [X ] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [NA ] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [X ] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - X If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [ X] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [X ] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x ] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,6 +156,7 @@ To learn more about the docker pull rate limits and the open source software pro
 | [Staramr](https://hub.docker.com/r/staphb/staramr/) <br/> [![docker pulls](https://badgen.net/docker/pulls/staphb/staramr)](https://hub.docker.com/r/staphb/staramr) | <ul><li>0.5.1</li><li>0.7.1</li><li>0.8.0</li></ul> | https://github.com/phac-nml/staramr |
 | [TBProfiler](https://hub.docker.com/r/staphb/tbprofiler/) <br/> [![docker pulls](https://badgen.net/docker/pulls/staphb/tbprofiler)](https://hub.docker.com/r/staphb/tbprofiler) | <ul><li>4.3.0</li><li>4.4.0</li></ul> | https://github.com/jodyphelan/TBProfiler |
 | [TipToft](https://hub.docker.com/r/staphb/tiptoft/) <br/> [![docker pulls](https://badgen.net/docker/pulls/staphb/tiptoft)](https://hub.docker.com/r/staphb/tiptoft) | <ul><li>1.0.0</li><li>1.0.2</li></ul> | https://github.com/andrewjpage/tiptoft |
+| [Tostadas](https://hub.docker.com/r/staphb/tostadas/) <br/> [![docker pulls](https://badgen.net/docker/pulls/staphb/tostadas)](https://hub.docker.com/r/staphb/tostadas) | <ul><li>1.0.0</li></ul> | https://github.com/CDCgov/tostadas |
```
Comment (erinyoung) [inline README.md:159]: This looks great! Can you also add your github handle at the bottom of the readme?

File: tostadas/1.0.0/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
+FROM nfcore/base:1.13.3
+LABEL authors="Cole and Ankush Gupta" \
+      description="Docker image containing all software requirements for mamba env"
```
Comment (erinyoung) [inline tostadas/1.0.0/Dockerfile:3]: Could you add more labels? These are from https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile

```
LABEL base.image="ubuntu:focal"
LABEL dockerfile.version="1"
LABEL software="SoftwareName"
LABEL software.version=$SOFTWARENAME_VER
LABEL description="This software does X, Y, AND Z!"
LABEL website="https://github.com/StaPH-B/docker-builds"
LABEL license="https://github.com/StaPH-B/docker-builds/blob/master/LICENSE"
LABEL maintainer="FirstName LastName"
LABEL maintainer.email="my.email@email.com"
```

File: tostadas/1.0.0/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,27 @@
+FROM nfcore/base:1.13.3
```
Comment (erinyoung) [inline tostadas/1.0.0/Dockerfile:1]: Is there a reason you're using nfcore/base instead of micromamba?

**Comment by erinyoung** [comment]
We won't be able to accept this dockerfile without both an 'app' and a 'test' layer. Please see our [template](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile) or any of the recent dockerfiles for examples on how this looks. Since this is a pipeline, it may be too long for github actions, but I think we should attempt it. Are there files somewhere that we could use?

We need you to add additional labels, this helps us maintain them.

I can't find any changes to [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md), were they never commited? You checked the task above, so I'm assuming you did it.

Your readme is also a little sparse.

Would you like to talk over a google meet?

**Comment by erinyoung** [comment]
Thank you for contributing to this resource! I don't think anything too drastic needs to happen, but there are some things that need to be added to get this accepted.

Don't hesitate to talk to me via SLACK, email, or you can ask questions here.

File: tostadas/1.0.0/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+LABEL software.version=$TOSTADAS_VER
+LABEL description="Image for the TOSTADAS: Toolkit for Open Sequence Triage, Annotation and DAtabase Submission pipeline"
+LABEL website="https://github.com/CDCgov/tostadas"
+LABEL license="https://github.com/CDCgov/tostadas/LICENSE"
+LABEL maintainer="Ankush Gupta"
+LABEL maintainer.email="ankushkgupta@deloitte.com"
+LABEL maintainer2="Kyle O'Connell"
+LABEL maintainer2.email=""
+LABEL maintainer3="Cole Tindall"
+LABEL maintainer3.email=""
```
Comment (erinyoung) [inline tostadas/1.0.0/Dockerfile:18]: I've added on the additional maintainers labels, but I don't know the emails for Kyle or Cole.

**Comment by kyleoconnell** [comment]
@erinyoung can you please take another look and see if this is ready to merge?

File: tostadas/0.2.0-beta/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+LABEL software.version=$TOSTADAS_VER
+LABEL description="Image for the TOSTADAS: Toolkit for Open Sequence Triage, Annotation and DAtabase Submission pipeline"
+LABEL website="https://github.com/CDCgov/tostadas"
+LABEL license="https://github.com/CDCgov/tostadas/LICENSE"
+LABEL maintainer="Ankush Gupta"
+LABEL maintainer.email="ankushkgupta@deloitte.com"
+LABEL maintainer2="Kyle O'Connell"
+LABEL maintainer2.email=""
+LABEL maintainer3="Cole Tindall"
+LABEL maintainer3.email=""
```
Comment (erinyoung) [inline tostadas/0.2.0-beta/Dockerfile:28]: This is missing some email information

**Comment by erinyoung** [comment]
It looks like some of the maintainer emails are missing. Do they need to be removed?

Also, could you look over tostadas/0.2.0-beta/README.md to make sure it makes sense?

- I updated the file paths to reflect the 0.2.0-beta version
- I updated README.md to reflect the 0.2.0-beta version
- I added Tostadas license to Program_licenses.md
- I deleted the other unused files
- I added text to tostadas/0.2.0-beta/README.md

File: tostadas/0.2.0-beta/Dockerfile (hunk 5 above)
Comment (ankushkgupta2) [reply tostadas/0.2.0-beta/Dockerfile]: Just pushed these changes to add the email information for Kyle and Cole

**Comment by ankushkgupta2** [comment]
Just checked the README.md and the other changes you mentioned. Everything looks perfect! Let us know if it is ready to merge with the most recent changes or if anything else is needed @erinyoung @kyleoconnell

**Review by erinyoung — APPROVED** [review]
Looks good! 

I'm going to merge this PR and push this image to both dockerhub and quay. Unless there's some sort of error, you should be able to pull it from there within the hour.

You should be able to access this container using the 'latest' tag as well as the '0.2.0-beta' tag.

**Pull request merged**